PhpUnit/AbstractExtensionTestCase.php

```diff
@@ -44,13 +45,22 @@ protected function setUp()
      * Call this method from within your test after you have (optionally) modified the ContainerBuilder for this test
      * ($this->container).
      *
-     * @param array $specificConfiguration
+     * If your extension(s) implements \Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface, you may
+     * set $withPrependInvocation to TRUE to invoke prepend() method prior to load() method of your extension.
+     *
+     * @param array $configurationValues
+     * @param bool $withPrependInvocation
      */
-    protected function load(array $configurationValues = array())
+    protected function load(array $configurationValues = array(), $withPrependInvocation = false)
```

Comment:
Careful, it could introduce BC breaks. If prepend method should be invoked always, that could cause some problems to current users of this library, so version of this library should be increased to 3.x.